### DISCUSSION TITLE
Running one flow at the time

### DISCUSSION THREAD

**Discussion by NoName115** [PrefectHQ/prefect#3469] [Q&A] [answered]
Hi, I would like to ask if there's some way to create new Executor or Agent or something else, how I would ensure that the Agent will run just one flow at the time, wait until it's finished and then run a next one?

Thanks a lot for you answer.

**Comment by cicdw** [comment, marked as answer]
Hi @NoName115 - if you use Prefect Cloud, you can limit the allowable concurrency of flow runs by [label](https://docs.prefect.io/orchestration/agents/overview.html#flow-affinity-labels).  In your case, you'd pick a label that is unique to your Agent and then set the concurrency limit of that label to 1.

I'm glad you asked, because it appears we have forgotten to properly document this feature!
